cli/command/service/helpers.go

```diff
@@ -15,20 +15,20 @@ import (
 
 // waitOnService waits for the service to converge. It outputs a progress bar,
 // if appropriate based on the CLI flags.
-func waitOnService(ctx context.Context, dockerCli *command.DockerCli, serviceID string, quiet bool) error {
+func waitOnService(ctx context.Context, cli command.Cli, serviceID string, quiet bool) error {
```

Comment:
I would keep using `dockerCli` as variable name (because we do it elsewhere and to not *shadow* the `cli` package)